### PR TITLE
Update the Azure.Core changelog to be more consise and complete for the beta.8

### PR DESCRIPTION
**Pull request by ahsonkhan**

### CODE REVIEW

**Comment by ahsonkhan** [comment]
Reviewing the commits since the last release: https://github.com/Azure/azure-sdk-for-cpp/compare/a8649822084ffa518484c31870986306d404f240...fb000d55b5e3108bfa3c548348a014c609821119

**Review by antkmsft — APPROVED** [review]

**Comment by ahsonkhan** [comment]
@antkmsft want to add `ADFS` as a new future in identity?
https://github.com/Azure/azure-sdk-for-cpp/pull/1947#discussion_r603698460

**Comment by ahsonkhan** [comment]
@vhvb1989 , we should add this to new feature section since it is meaningful to the customer, agreed?
https://github.com/Azure/azure-sdk-for-cpp/pull/2026

Also, we need a bug fix section for https://github.com/Azure/azure-sdk-for-cpp/pull/1500 once it is ready to ship.

**Comment by vhvb1989** [comment]
> @vhvb1989 , we should add this to new feature section since it is meaningful to the customer, agreed?
> #2026
> 
> Also, we need a bug fix section for #1500 once it is ready to ship.

I didn´t add it because it is expected to be transparent for end-users.  I must admit that, whit the time, and each release, I am caring less and less abut the changelog. For many reasons, I think it's currently giving us more issues than benefits. So, I am happy with anything in it or not in it :)

**Review by vhvb1989 — APPROVED** [review]

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## 1.0.0-beta.5 (2021-04-07)
 
+### New Features
+
+- Add Active Directory Federation Service (ADFS) support to `ClientSecretCredential`.
```
Comment (ahsonkhan) [inline sdk/identity/azure-identity/CHANGELOG.md:7]: @antkmsft can you take another look at this PR, specifically identity CL changes?

**Pull request merged**